runtime/vm/createramclass.cpp

```diff
@@ -2804,10 +2804,10 @@ internalCreateRAMClassFromROMClass(J9VMThread *vmThread, J9ClassLoader *classLoa
 					}
 
 					packageUTF = J9UTF8_DATA(J9ROMCLASS_CLASSNAME(packageRomClass));
-					pkgNameLength = packageNameLength(packageRomClass);
+					pkgNameLength = (U_32) packageNameLength(packageRomClass);
```

Comment:
Changing the declared type to `UDATA` would remove the need for these two changes.

Reply:
I started down the same route but `findModuleForPackage` takes a U_32, and then truncates it to a U_16.  Fixing that is outside the scope of unblocking the windows builds.

Reply:
I was only suggesting the one-line change based on the other uses of `packageNameLength` I could find, but I agree this is close enough.